### PR TITLE
UBER-1073

### PR DESCRIPTION
**Pull request by haiodo**
Fix flicking of UI

# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/ui/src/components/Component.svelte [hunk 1]
```diff
@@ -20,20 +20,31 @@
...
   export let shrink: boolean = false
   export let showLoading = true
   export let inline: boolean = false
   export let disabled: boolean = false
 
-  $: component = is != null ? getResource(is) : Promise.reject(new Error('is not defined'))
+  let _is: any
+  let _props: any
+
+  $: if (!deepEqual(_is, is)) {
```
Comment (BykhovDenis) [inline packages/ui/src/components/Component.svelte:36]: I'm not sure that we need deep equal here)

File: plugins/view-resources/src/components/list/ListCategories.svelte [hunk 2]
```diff
@@ -179,6 +180,24 @@
     }
   }
 
+  let oldClass: any
+  let oldConfig: any
+  let oldConfigurations: any
+  $: {
+    console.log(
```
Comment (BykhovDenis) [inline plugins/view-resources/src/components/list/ListCategories.svelte]: Extra

File: plugins/tracker-resources/src/components/issues/IssuePresenter.svelte [hunk 3]
```diff
@@ -30,22 +30,15 @@
...
-    } else {
-      currentProject = value?.$lookup?.space ?? projects?.get(value?.space)
-      spaceQuery.unsubscribe()
-    }
+    currentProject = $activeProjects.get(value?.space)
   }
 
   $: title = currentProject ? `${currentProject.identifier}-${value?.number}` : `${value?.number}`
+
+  $: console.log('CPC:', value.$lookup?.space, currentProject)
```
Comment (BykhovDenis) [inline plugins/tracker-resources/src/components/issues/IssuePresenter.svelte]: extra

**Review by BykhovDenis — APPROVED** [review]

File: plugins/view-resources/src/components/list/ListCategories.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/view-resources/src/components/list/ListCategories.svelte]: Fixed.

File: plugins/tracker-resources/src/components/issues/IssuePresenter.svelte (hunk 3 above)
Comment (haiodo) [reply plugins/tracker-resources/src/components/issues/IssuePresenter.svelte]: Fixed.

File: packages/ui/src/components/Component.svelte (hunk 1 above)
Comment (haiodo) [reply packages/ui/src/components/Component.svelte:36]: Any other way to check?

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**